README.md

```diff
@@ -26,7 +26,7 @@ For a curated list of learning resources please check out [desireevl's repo](htt
 ## Quantum full-stack libraries
 
 **C++**
-- [staq](https://github.com/softwareqinc/staq) - [Full stack quantum processing toolkit](https://arxiv.org/abs/1912.06070).
+- [staq](https://github.com/softwareqinc/staq) - [Full stack quantum processing toolkit](https://iopscience.iop.org/article/10.1088/2058-9565/ab9359).
```

Comment:
Looks like this change would introduce an article behind a paywall. Why not leave the arXiv version?

Reply:
sounds good, I'll then update the arXiv with the link to the journal